templates/components/signature.build.hbs

```diff
@@ -1,11 +1,7 @@
 <div v-if="description" class="help-block description">\{{ description }}</div>
-<template v-if="readonly">
-  <div class="field-signature focus-outline" tabindex="0">
-    <img :src="value" alt="signature image" />
-  </div>
-</template>
-<template v-else>
-  <div v-show="isEditable" class="signature-editor">
+
```

Comment:
Can you please remove new empty blank line

Reply:
done changes